### ISSUE TITLE
[Issue]: Screen wont turn off

### ISSUE THREAD

**Issue by IAJaay** [SamSprung/SamSprung-TooUI#239]
### Issue Description

**Device**  Samsung Z Flip 3
**What Happens** The screens (cover and main screen) wont turn off after a while as should but stays lit. The screens seems to be powered on as long as user does not intervene. Pressing the side button to turn off the screen seems to work okay. Not sure of double tapping though (however if I remember correctly the double tapping propably works okay). The issue persists every time the screens are used and not closed by manual user action.
**When the Issue Arises**  Seemingly pops up randomly without any spesific cause.
**What Helps** Removing tooui app data fixes the issue. Cache removal does not help. Also double tapping the samsprung icon from cover screen as instructed in the guide seemed to fix the issue.

If this issue is related to the already mentioned issue in the guide _"Notice: Due to an Android 13 bug, touch is not passed to the stock launcher when TooUI is minimized. Double tap the SamSprung icon to temporarily disable the launcher until the screen is turned off."_ , could you rephrase the guide notice so that readers gets that it relates to this sort of behaviour.

### Logcat Output

```shell
SamSprung 2.6.1 (Play Release) #87df9ad
samsung UPSIDE_DOWN_CAKE (14)
Secure Lock Screen

--------- beginning of main

07-27 08:39:22.535 9869 9869 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=falsecallback=android.app.Activity$$ExternalSyntheticLambda0@7b7e78c
07-27 10:10:15.382 9869 9869 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=falsecallback=android.app.Activity$$ExternalSyntheticLambda0@e86a472
07-27 10:10:38.906 9869 9869 W RecyclerView: No adapter attached; skipping layout
07-27 10:10:38.948 9869 9869 W Kumiho-Kumiho: isSupportedFormat: Format is not supported
07-27 10:10:38.990 9869 9869 W Activity: Can request only one set of permissions at a time
07-27 10:10:39.061 9869 9880 W prung:SamSprung: Reducing the number of considered missed Gc histogram windows from 278 to 100
```

### Guide Confirmation

- [X] I have reviewed the guide and included valid logcat content

**Comment by AbandonedCart** [comment]
Open the app from the main screen. Expand drawer options. Set your screen timeout.
